<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>666</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>666</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GameWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>666</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>666</width>
     <height>800</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="GameArea" name="widgetGameArea" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>350</width>
      <height>700</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widgetBGD" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>666</width>
      <height>800</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>666</width>
      <height>800</height>
     </size>
    </property>
   </widget>
   <widget class="NextArea" name="widgetNextArea" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>170</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(247, 247, 247);</string>
    </property>
    <property name="text">
     <string>NEXT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widgetN" native="true">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>190</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widgetG" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>370</width>
      <height>720</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widgetG_2" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>350</width>
      <height>700</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widgetN_2" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>170</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <zorder>widgetBGD</zorder>
   <zorder>label</zorder>
   <zorder>widgetN</zorder>
   <zorder>widgetG</zorder>
   <zorder>widgetG_2</zorder>
   <zorder>widgetGameArea</zorder>
   <zorder>widgetN_2</zorder>
   <zorder>widgetNextArea</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NextArea</class>
   <extends>QWidget</extends>
   <header>nextarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameArea</class>
   <extends>QWidget</extends>
   <header>gamearea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
